mlflow/pytorch/__init__.py

```diff
@@ -539,14 +649,44 @@ def load_model(model_uri, **kwargs):
     .. code-block:: python
         :caption: Example
 
+        import warnings
+        import os
```

Comment:
```suggestion
```

Can we remove these because they don't seem to  be used?

Reply:
As cut-n-paste leftover over from [here](https://github.com/dmatrix/mlflow-tests/blob/master/py/mlflow/models/pytorch/log_model.py). :)